contrib/confluentinc/confluent-kafka-go/kafka/kafka_test.go

```diff
@@ -149,6 +150,13 @@ func TestConsumerChannel(t *testing.T) {
 		assert.Equal(t, ext.SpanKindConsumer, s.Tag(ext.SpanKind))
 		assert.Equal(t, "kafka", s.Tag(ext.MessagingSystem))
 	}
+	for _, msg := range []*kafka.Message{msg1, msg2} {
+		p, ok := datastreams.PathwayFromContext(datastreams.ExtractFromCarrier(context.Background(), NewMessageCarrier(msg)))
+		assert.True(t, ok)
+		expected, _, _ := tracer.SetDataStreamsCheckpoint(context.Background(), "group:"+testGroupID, "direction:in", "topic:"+testTopic, "type:kafka")
+		assert.NotEqual(t, 0, expected.GetHash())
+		assert.Equal(t, p.GetHash(), expected.GetHash())
```

Comment:
assert expected than actual (looks like there's a few more throughout)
```suggestion
		assert.Equal(t, expected.GetHash(), p.GetHash())
```